core/src/test/java/org/elasticsearch/index/query/PercolatorQueryBuilderTests.java

```diff
@@ -75,6 +79,16 @@ private PercolatorQueryBuilder doCreateTestQueryBuilder(boolean indexedDocument)
         }
     }
 
+    /**
```

Comment:
the bytes are not the same right? I guess it depends on how we compare bytes references.... would be nice to get around it so we don't need this odd method at all :)